### PR TITLE
scdoc: tweak font size and distances

### PR DESCRIPTION
**Pull request by telephon**
These are some quick tweaks to improve the new help file rendering a little.

On the long run, it would be a good idea to clean up the css

### CODE REVIEW

File: HelpSource/scdoc.css [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-    font-family: Arial, Helvetica, sans-serif;
+    font-family: Helvetica, Arial, sans-serif;
     font-size: 10pt;
     color: black;
     background: white;
 /*    max-width: 60em;*/
 /*    margin-left: 20em;*/
-    padding: 0;
+    width: auto;
+    padding-left: 1.5em;
```
Comment (snappizz) [inline HelpSource/scdoc.css]: `width: auto` adds a horizontal scrollbar to some pages (e.g. the Array helpfile for me). `padding-left: 1.5em` adds whitespace to the left of the top bar so it looks "broken off."

File: HelpSource/scdoc.css [hunk 2]
```diff
@@ -15,10 +17,9 @@ html {
 }
 
 div.contents {
-    margin: 1em;
     padding-left: 0.3em;
     padding-right: 0.3em;
-    padding-top: 1.5em;
+    padding-top: 1.4em;
```
Comment (snappizz) [inline HelpSource/scdoc.css]: i'm not big on this. the top "SuperCollider 3.9dev Classes" looks cramped against the top bar.

File: HelpSource/scdoc.css (hunk 1 above)
Comment (telephon) [inline HelpSource/scdoc.css]: yes, I see. The problem is that without it, the method names are stuck to the left, abe the stars for the class methods are half-hidden behind the frame.

if you know another way, let me know.

Comment (telephon) [inline HelpSource/scdoc.css]: actually, now it looks different again. So never mind, some rendering mixup.

File: HelpSource/scdoc.css [hunk 3]
```diff
@@ -15,7 +15,6 @@ html {
 }
 
 div.contents {
-    margin: 1em;
```
Comment (snappizz) [inline HelpSource/scdoc.css]: sorry, i think i commented on the wrong line. this is what's making it cramped. i recommend leaving this in.

**Comment by telephon** [comment]
… here you are.

File: HelpSource/scdoc.css [hunk 4]
```diff
@@ -356,7 +357,7 @@ a.method-name {
 
 .method {
     margin-left: 2em;
-    margin-bottom: 2em;
+    margin-bottom: 3.5em;
```
Comment (snappizz) [inline HelpSource/scdoc.css]: sorry for being so picky.... i think this is a little too sparse. `2.5em`?

Comment (telephon) [inline HelpSource/scdoc.css]: yes, sure! Does this PR still make sense? You can just use my suggestions in your redesign, if you like, and close it.

**Pull request closed without merging**